packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -50,6 +50,8 @@
     let statusLoaded: boolean = false
     let overviewLoaded: boolean = false
     let openedQuestionIndex: number = -1
+    let isUpdatingVotedAnswerValues: boolean = false
+    let lastAction: 'vote' | 'stopVote'
```

Comment:
We have a [`GovernanceAction`](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/core/wallet/enums/governance-action.enum.ts) enum we should use here.